pkg/feature/domain/evaluation.go

```diff
@@ -158,6 +160,9 @@ func evaluate(
 		}
 		// VariationId is used to check if prerequisite flag's result is what user expects it to be.
 		flagVariations[f.Id] = variation.Id
+		if targetTag != "" && !tagExist(f.Tags, targetTag) {
```

Comment:
It would be nice to comment on how the tag works here.

E.g.
```go
// When the tag is set in the request, it will evaluate only the flags that match the tag configured when the flag was created on the dashboard.
// When empty, it will evaluate all the flags in the environment. 
```

Reply:
@cre8ivejp 
Thx!
I added a comment!